From 002fb66f78e42b78a256c445190858174738b91a Mon Sep 17 00:00:00 2001
From: manchandavishal <manchandavishal143@gmail.com>
Date: Sat, 5 Oct 2019 10:53:34 +0000
Subject: [PATCH] Use Horizon project template for django jobs

Horizon defined a project template 'horizon-non-primary-django-jobs'
for django jobs. This patch use that template to run django jobs
here and also fixes failed test cases for django2.2 support.
For information please refer [1]

[1] https://review.opendev.org/#/c/681969/

Change-Id: I3ab81402633b9abe39e49c3921585db992050540
---
 .zuul.yaml                                       |  5 +----
 heat_dashboard/test/tests/content/test_stacks.py | 11 +++--------
 tox.ini                                          | 13 +++----------
 3 files changed, 7 insertions(+), 22 deletions(-)

--- a/heat_dashboard/test/tests/content/test_stacks.py
+++ b/heat_dashboard/test/tests/content/test_stacks.py
@@ -353,7 +353,7 @@ class StackTests(test.TestCase):
         self.assertTemplateUsed(res, 'project/stacks/create.html')
 
         # ensure the fields were rendered correctly
-        if (1, 10) <= django.VERSION < (2, 1):
+        if (1, 11) <= django.VERSION:
             pattern = ('<input class="form-control" '
                        'id="id___param_public_string" '
                        'name="__param_public_string" type="text" required/>')
@@ -511,7 +511,7 @@ class StackTests(test.TestCase):
         self.assertTemplateUsed(res, 'project/stacks/create.html')
 
         # ensure the fields were rendered correctly
-        if (1, 10) <= django.VERSION < (2, 1):
+        if (1, 11) <= django.VERSION:
             input_str = ('<input class="form-control" '
                          'id="id___param_param{0}" '
                          'name="__param_param{0}" type="{1}" required/>')
@@ -522,16 +522,11 @@ class StackTests(test.TestCase):
         self.assertContains(res, input_str.format(3, 'text'), html=True)
         self.assertContains(res, input_str.format(4, 'text'), html=True)
 
-        if (1, 11) <= django.VERSION < (2, 1):
+        if (1, 11) <= django.VERSION:
             input_str_param2 = ('<input type="number" name="__param_param2" '
                                 'autocomplete="off" '
                                 'required class="form-control" '
                                 'id="id___param_param2" />')
-        elif (1, 10) <= django.VERSION < (1, 11):
-            input_str_param2 = ('<input autocomplete="off" '
-                                'class="form-control" id="id___param_param2" '
-                                'name="__param_param2" type="number" '
-                                'required />')
         else:
             input_str_param2 = ('<input autocomplete="off" '
                                 'class ="form-control" '
